content/2021-11-19-brigade-2-cuts-first-release-candidate.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Brigade 2 Cuts First Release Candidate"
+slug: "2021-11-19-brigade-2-cuts-first-release-candidate"
+description: "Brigade 2 has cut its first release candidate"
+date: "2021-11-19"
+author: "Kent Rancourt"
+summary: Brigade 2 has cut its first release candidate.
+---
+
+We have exciting news, Brigadiers!
+
+Yesterday afternoon, maintainers cut the first Brigade 2 release candidate. This release candidate ([v2.0.0-rc.1](https://github.com/brigadecore/brigade/releases/tag/v2.0.0-rc.1)) also marks Brigade 2's transition from serving the v2 beta API to serving the _stable_ v2 API. With this new level of stability, there's never been a better time to get started with Brigade 2, so please do consider heading over to our recently refreshed [QuickStart](https://v2--brigade-docs.netlify.app/intro/quickstart/) and taking us for a spin.
+
+The transition to the stable API means previous versions of gateways and other peripherals are not compatible with this release candidate, but as of this morning, we have _also_ transitioned every one of those projects to the stable API and cut a beta release of each. (They each currently remain at a beta level of maturity despite Brigade 2 itself having advanced to a release candidate.)
+
+Barring any major surprises, we anticipate the only PRs merged between now and December 1 will be release-oriented chores, so we're targeting that date for our GA release -- just in time for the holiday season!
+
+Following the Brigade 2 GA release, maintainers will begin putting the finishing touches on all our peripherals and will release each of those individually as they reach maturity, with the GitHub gateway almost certainly being the first.
+
+Last, but certainly not least, we also want to thank [Jorge Arteiro](https://github.com/jorgearteiro) for recently stepping up to start reviewing chore-type PRs across all our repos!
```

Comment:
I know that "chore" is a designation, but in this context it might be clearer to say:

Last, but certainly not least, we also want to thank [Jorge Arteiro](https://github.com/jorgearteiro) for recently starting to review PRs across all the Brigade repos.

Reply:
This is a tricky one...

Technically, maintainers should review everything. The idea of designating certain PRs as reviewable by non-maintainers is a recent thing we're trying to alleviate some bottlenecks. I think there is value in maintaining that distinction...

But I could be convinced otherwise.

Reply:
maybe "Last, but certainly not least, we also want to thank Jorge Arteiro for recently starting to assist in PR review across all the Brigade repos."

I'm trying to get away from the word "chore" since you already used it once in this short blog post, and since to a casual reader it may seem like you're diminishing the work he's doing (which isn't the impression you want to give).

Reply:
Yes... I like that!

Reply:
Fixed in https://github.com/brigadecore/blog/commit/021e94694b4438507085c17e00f00f3a260c001e